Traducir.Web/src/App/Components/SuggestionHistoryFilters.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { StringSuggestionState } from "../../Models/SOStringSuggestion";
+import { NonUndefinedReactNode } from "../NonUndefinedReactNode";
+
+export interface ISuggestionHistoryFiltersProps {
+    userid: number;
+}
+
+export default class Filters extends React.Component<ISuggestionHistoryFiltersProps> {
+    public render(): NonUndefinedReactNode {
+        return <div className="row text-center mt-3">
+            <div className="col d-none d-lg-block">
+                <div className="btn-group" role="group" aria-label="Basic example">
```

Comment:
would you mind adding some visual cue as to what filter is applied? changing the button color would work :)